app/upload/views.py

```diff
@@ -27,12 +27,14 @@ def upload_document(service_id):
 
     filename = request.form.get('filename')
 
+    sending_method = request.form.get('sending_method')
```

Comment:
Can remove the `.get` now that args are required? Make sure we fail loudly if those are not here

Reply:
nevermind, i tried this and it broke backwards compatibility to existing links as well as required code changes in document frontend.